### DISCUSSION TITLE
HTML Format extension isn't working

### DISCUSSION THREAD

**Discussion by SteveM2020** [phcode-dev/phoenix#1868] [Q&A]
I  installed the "HTML Format" extension. It isn't working. I have Brackets on a another computer with this extension working. I thought I might copy it over to the Phoenix editor, to see if it would work.

My questions are, will this break anything? Is there a better solution for fixing this problem?

Thank you,
Steve

**Comment by abose** [comment]
Closing as duplicate of https://github.com/phcode-dev/phoenix/issues/1869
